lib/cucumber/cli/project_initializer.js

```diff
@@ -0,0 +1,11 @@
+var ProjectInitializer = {
+  createSaneDefaultProject: function createSaneDefaultProject() { 
```

Comment:
How about just `createDefaultProject`? I don't think we need to have `sane` in the name (unless we also have the method `createInsaneDefaultProject`, which I don't think is necessary :smile:)